inbm-vision/node-agent/node/data_handler.py

```diff
@@ -54,8 +54,8 @@ def __init__(self, node_callback: inode.INode, config_callback: ConfigurationMan
         self._heartbeat_interval: Optional[int] = None
         self._nid: Optional[str] = None
         self._timer: Optional[HeartbeatTimer] = None
-        self._retry_limit: int = 0
-        self._retry_interval: int = 0
+        self._retry_count: int = 0
+        self._retry_limit: int = CONFIG_REGISTRATION_RETRY_LIMIT.default_value
         self._retry_timer: int = 0
```

Comment:
Changed this around as it was confusing.  The _retry_interval was actually the limit and the _retry_limit was actually the count.